front/assets/translations/en_US.json

```
@@ -1,5 +1,53 @@
 {
   "internal": {
     "translatedText": "This text is translated to English."
+  },
+  "profile": {
+    "info": {
+      "contact": "Contact",
+      "availabilities": "Availabilities",
+      "experiences": "Experiences",
+      "reviews": "Reviews",
+      "phoneNumber": "Phone number",
+      "email": "Email",
+      "references": "References",
+      "downloadResume": "Download resume",
+      "informations": "Informations",
```

(The word is uncountable in English)
```suggestion
      "information": "Information",
```